package.json

```diff
@@ -48,7 +50,8 @@
 		"serve": "node build/index.js",
 		"serve:forever": "node build/daemon.js",
 		"build-ts": "tsc",
-		"watch-ts": "tsc -w"
+		"watch-ts": "tsc -w",
+		"reformat-files": "./node_modules/.bin/prettier --write --require-pragma \"src/**/*.{ts,tsx,scss}\""
```

Comment:
the whole codebase is prettified, or at least it should be.  I believe you can remove the `--require-pragma`